README.md

```diff
@@ -462,6 +462,7 @@ Table of Contents
   * [reflect.run](https://reflect.run) - Codeless automated tests for web apps. Tests can be scheduled in-app or executed from a CI/CD tool. Each test run includes a full video recording along with console and network logs. The free tier includes an unlimited number of saved tests, with 25 test runs per month and up to 3 users.
   * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
   * [testingbot.com](https://testingbot.com/) — Selenium Browser and Device Testing, [free for Open Source](https://testingbot.com/open-source)
+  * [Testspace.com](https://testspace.com/) - Dashboard for publishing automated test results and a Framework for implementing manual tests as code using GitHub. [Free for Open Source](https://github.com/marketplace/testspace-com).
```

Comment:
Please add something like `the SaaS version is free for 450 requests a month`

Reply:
Added 450 results per month.